rafthttp/stream.go

```diff
@@ -116,6 +116,7 @@ func (cw *streamWriter) run() {
 				if m.Term > msgAppTerm {
 					cw.resetCloser()
 					heartbeatc, msgc = nil, nil
+					cw.r.ReportUnreachable(m.To)
```

Comment:
i will add a TODO for it. currently we close the connection and fall to unreachable status.